src/utils/__tests__/getPriceGuidance.test.ts

```diff
@@ -0,0 +1,50 @@
+import { getPriceGuidance } from "../getPriceGuidance"
+jest.mock("../../lib/metaphysics", () => ({
+  metaphysics: jest.fn(),
+}))
+
+const mockMetaphysics = require("../../lib/metaphysics").metaphysics
+
```

Comment:
Just checked other mockings of this, i think the issue is that you want the default rather than named export: `jest.mock("../../lib/metaphysics", () => ({ default: jest.fn() }))`

Reply:
Ahhh, thanks!